crates/proto/src/rr/rdata/opt.rs

```diff
@@ -186,6 +186,11 @@ impl OPT {
         &self.options
     }
 
+    /// The entire map of options
+    pub fn options_mut(&mut self) -> &mut HashMap<EdnsCode, EdnsOption> {
+        &mut self.options
+    }
+
```

Comment:
Should we just leave this out in favor of the more standard `AsMut` approach?

Reply:
Yeah I didn't know which direction to go there so I added both. I'm happy to mark `options` deprecated and remove this in favor of `AsRef`/`AsMut` if you're on board with that.